src/style.css

```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-weight: 300;
+  --background-color: #fff;
+}
+
+.search-bar {
+  display: flex;
+  align-items: center;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  background: var(--background-color);
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  margin-top: 20px;
+}
+
+.search-bar__big {
+  height: 70px;
+}
+
+.search-bar__small {
+  height: 42px;
+}
+
+.search-bar__logo--big {
+  width: 19px;
+  height: 19px;
+  padding-left: 26px;
+  padding-right: 17px;
+}
+
+.search-bar__logo--small {
```

Comment:
check code style best practices - https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#style-using-classes

you have two similar elements you can create one base class and one where you will override needed styles